lib/class-wp-theme-json-data-gutenberg.php

```diff
@@ -16,7 +16,7 @@ class WP_Theme_JSON_Data_Gutenberg {
 	 * Container of the data to update.
 	 *
 	 * @since 6.1.0
-	 * @var WP_Theme_JSON
+	 * @var WP_Theme_JSON_Gutenberg
```

Comment:
I would probably leave similar comments in place, as it makes it easier to diff backports.

Reply:
Good point I reverted this change.